src/main/java/org/wildfly/security/auth/realm/FileSystemSecurityRealm.java

```diff
@@ -347,20 +347,26 @@ public SupportLevel getEvidenceVerifySupport(final Class<? extends Evidence> evi
             List<Credential> credentials = loadCredentials();
             for (Credential credential : credentials) {
                 if (credential.canVerify(evidenceType, algorithmName)) {
+                    ElytronMessages.log.tracef("FileSystemSecurityRealm - evidence verification SUPPORTED: type = [%s]  algorithm = [%s]  credentials = [%d]", evidenceType, algorithmName, credentials.size());
```

Comment:
Well "a lot" would mean at most, like, 20 or so, which is really very little in the grand scheme.  A subcategory "org.wildfly.security.foo.bar" is also generally configurable at its parent category of "org.wildfly.security.foo".

That said I'm fine either way, it just seems wrong to put what is essentially a category into the actual log message.